joomla/joomla-cms
# [4.0] Implement condition query for frontend modules/plugins

### Summary of Changes
Change hardcoded state calls to the new condition constants



### Testing Instructions
- Create mod_stats or mod_related_items modules and  check the numbers
- Search content in frontend
- Check pagenav in article view
- They should work now like in J! 3 again.
